Client2.py

```diff
@@ -0,0 +1,45 @@
+"""Client Class File"""
+
+import socket
+
+
+class Client:
+    """comment"""
```

Comment:
Please add proper comments about the class. The comments should describe what this class is and how it works.